<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xiaozhi-PyQt-Fluent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="navigation_layout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="SegmentedWidget" name="nav_segment">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="mainPageLayout">
       <item>
        <widget class="CardWidget" name="status_card">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="card_layout">
          <item>
           <widget class="QLabel" name="status_label">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #2196F3;
                     padding: 10px;
                     background-color: #E3F2FD;
                     border-radius: 8px;</string>
            </property>
            <property name="text">
             <string>Áä∂ÊÄÅ: Êú™ËøûÊé•</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="emotion_label">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string>margin: 15px;</string>
            </property>
            <property name="text">
             <string>üòä</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tts_text_label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>ÂæÖÂëΩ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="mic_visualizer_card">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>72</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="mic_visualizer_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>18</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>18</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="audio_control_stack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="volume_page">
             <layout class="QHBoxLayout" name="volume_layout">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="PushButton" name="mute">
                <property name="minimumSize">
                 <size>
                  <width>72</width>
                  <height>34</height>
                 </size>
                </property>
                <property name="text">
                 <string>Èü≥Èáè</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Slider" name="volume_scale">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="mic_page">
             <layout class="QVBoxLayout" name="mic_page_layout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="mic_visualizer_widget" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-radius: 8px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="button_layout">
         <item>
          <widget class="PushButton" name="manual_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>Êåâ‰ΩèÂêéËØ¥ËØù</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="abort_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>ÊâìÊñ≠ÂØπËØù</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="auto_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>ÂºÄÂßãÂØπËØù</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="mode_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>ÊâãÂä®ÂØπËØù</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="iotPage">
      <layout class="QVBoxLayout" name="historyPageLayout">
       <item>
        <widget class="CardWidget" name="history_card">
         <layout class="QVBoxLayout" name="history_layout">
          <item>
           <widget class="QLabel" name="history_title">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ÊöÇÊó†IOTËÆæÂ§á</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingPage">
      <layout class="QVBoxLayout" name="settingPageLayout">
       <item>
        <widget class="CardWidget" name="setting_card">
         <layout class="QVBoxLayout" name="setting_layout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="setting_title">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ÂèÇÊï∞ÈÖçÁΩÆ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="bottomMargin" stdset="0">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="wakeWordEnableLayout">
            <item>
             <widget class="QLabel" name="wakeWordEnableLabel">
              <property name="text">
               <string>ÂêØÁî®Âî§ÈÜíËØçÂî§ÈÜí:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="SwitchButton" name="wakeWordEnableSwitch">
              <property name="onText">
               <string>ÂºÄ</string>
              </property>
              <property name="offText">
               <string>ÂÖ≥</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_WakeEnable">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="wakeWordsEditLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="wakeWordsLabel">
              <property name="text">
               <string>Âî§ÈÜíËØç (Â§ö‰∏™Âî§ÈÜíËØçËØ∑Áî®Ëã±ÊñáÈÄóÂè∑ ',' ÂàÜÈöî):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="LineEdit" name="wakeWordsLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_Group1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="deviceIdLabel">
              <property name="text">
               <string>Device ID:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="LineEdit" name="deviceIdLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="websocketUrlLabel">
              <property name="text">
               <string>APIÊé•Âè£Âú∞ÂùÄ:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="wsUrlContainerWidget" native="true">
              <layout class="QHBoxLayout" name="wsUrlInnerLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="ComboBox" name="wsProtocolComboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="LineEdit" name="wsAddressLineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="websocketTokenLabel">
              <property name="text">
               <string>Access Token:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="LineEdit" name="wsTokenLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_Group2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="PushButton" name="saveSettingsButton">
            <property name="text">
             <string>‰øùÂ≠òËÆæÁΩÆ</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_Bottom">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>8</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Pivot</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SegmentedWidget</class>
   <extends>Pivot</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 